### PR TITLE
feat: Hotjar tracking integration to trigger survey within ORA iframe

### PR DESCRIPTION
**Pull request by jszewczulak**
**TL;DR -** Manual Hotjar tracking integration to trigger survey within ORA xblock iframe. 

JIRA: [AU-1471](https://2u-internal.atlassian.net/browse/AU-1471)

**Developer Checklist**

- [ ] Reviewed the [release process](https://github.com/openedx/edx-ora2/blob/master/.github/release_process.md)
- [ ] Translations and JS/SASS compiled
- [ ] Bumped version number in [openassessment/\_\_init\_\_.py](https://github.com/openedx/edx-ora2/blob/master/openassessment/__init__.py#L4) and [package.json](https://github.com/openedx/edx-ora2/blob/master/package.json#L3)

**Testing Instructions**

[ How should a reviewer test this PR? ]

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @openedx/content-aurora

### CODE REVIEW

File: openassessment/templates/openassessmentblock/base.html [hunk 1]
```diff
@@ -5,6 +5,17 @@
   If the new views are not enabled, `.ora-view` will be hidden, otherwise, `.ora-legacy-view`
   will be hidden.
 -->
+<!-- Hotjar Tracking Code for https://www.edx.org/ -->
+<script>
+    (function(h,o,t,j,a,r){
```
Comment (leangseu-edx) [inline openassessment/templates/openassessmentblock/base.html:10]: Question, are we tracking all of the open assessment?

Comment (jszewczulak) [reply openassessment/templates/openassessmentblock/base.html:10]: Yes, hotjar integration is for all ORAs, then we have logic in views to dispatch event under certain conditions to render survey widget.

Comment (leangseu-edx) [reply openassessment/templates/openassessmentblock/base.html:10]: Wouldn't that make sense to conditionally having hotjar include? If we only need it for survey, I don't see the reason to have it connect to all of the open assessment.

Comment (jszewczulak) [reply openassessment/templates/openassessmentblock/base.html:10]: Again, this is just hotjar platform integration for ORA iframe (we already have full integration in LMS). While we do want to dispatch survey event conditionally, we don't want to limit ourselves to survey only. Hotjar offers other services and we don't want to refer to code change and deployment each time we want to configure on Hotjar. Tat would be anti-pattern.

Comment (leangseu-edx) [reply openassessment/templates/openassessmentblock/base.html:10]: I don't think it is anti-pattern to just do what needed to be done. If there isn't a feature require hotjar in the roadmap or near future, it seems over kill to connect hotjar to all open assessment.

**Review by mattcarter — APPROVED** [review]
Approved

**Comment by kdmccormick** [comment]
I'm not happy to see more edX.org-specific tracking code added directly to an Open edX repository like this.

Since you've already added it, though, @jszewczulak will this load on sites that do not set the hotjar site id?
